plugins/github-actions/src/api/GithubActionsApi.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createApiRef } from '@backstage/core';
+import { Build, BuildDetails } from './types';
+
+export const githubActionsApiRef = createApiRef<GithubActionsApi>({
+  id: 'plugin.githubactions.service',
+  description: 'Used by the Github Actions plugin to make requests',
+});
+
+export type GithubActionsApi = {
+  listBuilds: (_entityUri: string) => Promise<Build[]>;
```

Comment:
why the underscores here? 😅

Reply:
Just an ugly leftover from the older code. Will be gone soon.